src/Uno.UI.FluentTheme/Resources/Priority02/MenuFlyout_themeresources.xaml

```diff
@@ -639,21 +639,21 @@
                             Grid.Column="0"
                             HorizontalAlignment="Left"
                             Margin="0">
-                            <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE8C1;" />
+                            <FontIcon FontFamily="{ThemeResource SymbolThemeFontFamily}" Glyph="&#xE8C1;" />
```

Comment:
Please prefix all the modified lines with a comment like the following. This helps to stop the fix getting regressed when the newest upstream resources are imported.

```suggestion
                            <!--UNO workaround: don't hardcode Segoe font-->
                            <FontIcon FontFamily="{ThemeResource SymbolThemeFontFamily}" Glyph="&#xE8C1;" />
```